plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/GitChangesHandler.java

```diff
@@ -59,26 +69,42 @@ public GitChangesHandler(
     this.projectExplorerPresenterProvider = projectExplorerPresenterProvider;
     this.multiPartStackProvider = multiPartStackProvider;
 
+    eventBus.addHandler(
+        EditorOpenedEvent.TYPE,
+        event -> {
+          if (((File) event.getFile()).getVcsStatus() != MODIFIED) {
+            return;
+          }
+          VcsChangeMarkerRender render =
```

Comment:
Check if not null

Reply:
Why? `getVcsChangeMarkersRender` method is not markes as `Nullable`.

Reply:
To initialized render it is used the snippet of code:
```
.findAny()
 .ifPresent()
...
```
So, it might be null.

Reply:
Reworked to creation of new ruler in https://github.com/eclipse/che/pull/6054/commits/8c22aa0bd8d5eab585a8d12715877d758f78cb9b
It can't be null now